enhancements/cert-manager/cert-manager-network-policies.md

```
@@ -0,0 +1,413 @@
+      - ports:
+        - protocol: TCP
+          port: 6443
+      ingress:
+      - ports:
+        - protocol: TCP
+          port: 10250 # Webhook port for API server
+        - protocol: TCP
+          port: 9402  # Metrics port
+    ```
+
+4.  **Allow `cert-manager-cainjector` Traffic:** This policy allows the CA injector to communicate with the API server and expose its metrics.
```

markdownlint-cli2 (0.17.2)

Spaces after list markers
Expected: 1; Actual: 2

(MD030, list-marker-space)